src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -133,7 +134,11 @@ const convertFilesystemToPartition = (filesystem: Filesystem): Partition => {
  * @param distribution blueprint distribution
  */
 const getLatestRelease = (distribution: Distributions) => {
-  return distribution === RHEL_9_BETA
+  return distribution === RHEL_10_BETA
+    ? (RHEL_10_BETA as Distributions)
+    : distribution.startswith('rhel-10')
+    ? RHEL_10
```

Comment:
There's no RHEL 10 option in API, should there be?

Reply:
right I see what you mean, hmmm